alicloud/provider.go

```diff
@@ -65,6 +65,7 @@ func Provider() terraform.ResourceProvider {
 			"alicloud_ram_policies":         dataSourceAlicloudRamPolicies(),
 			"alicloud_security_groups":      dataSourceAlicloudSecurityGroups(),
 			"alicloud_security_group_rules": dataSourceAlicloudSecurityGroupRules(),
+			"alicloud_rds_instances":        dataSourceAlicloudRdsInstances(),
```

Comment:
In order to consistent with rds resource, please change 'alicloud_rds_instances' to 'alicloud_db_instances'

Reply:
datasource name has been changed